index.Rmd

```diff
@@ -76,6 +65,47 @@ The original source material for this course is found on the
 [GitHub Actions]: https://github.com/features/actions
 [`r-rse`]: https://github.com/merely-useful/r-rse
 
+## Target audience
+
+While the learning content of this book targets the specific learning personas
+described in FIXME, we wrote this book with three audiences in mind:
+
+1. **Instructors** who would use this book as the basis for a course or workshop
+of their own. Throughout the book there are sections dedicated to this reader
+that give information or instructions on how to use or modify the material for
+the instructors specific context.
+2. **Learners in a class setting** who would be taking this course in a more
+formal class or workshop setting that is lead by an instructor(s). Much of the
```

Comment:
```suggestion
formal class or workshop setting that is lead by an instructor. Much of the
```

I don't know, we can keep the (s) but it just feels kind of awkward and maybe not necessary?
Alternate suggestion: 
```suggestion
formal class or workshop setting that is lead by an instructor or instructors. Much of the
```